copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+    "calendar_event_end": "{{$trigger.body.data.calendar_event_end}}",
+    "name": "{{$trigger.body.data.name}}",
+    "description": "{{$trigger.body.data.description}}"
+}
+```
+
+> Make sure that you are using the same key (here it's "item_read") as you set in node 5
+
+Query is empty
+
+
+***
+
+- Node 10 - "Condition"
+
+The flow is in the branch "item not found". So, if the action was to Update an item, then creating item is required (in the next node)
+
+![node 10](/copy-this-template-directory/directus_flow_4_10.png "node 10")
+
+Rules:
+```js
+{
+    "$trigger": {
+        "body": {
+            "action": {
+                "_eq": "update"
+            }
+        }
+    }
+}
+```
+
+
+***
+
+- Node 11 - "Create Data"
+
+![node 11](/copy-this-template-directory/directus_flow_4_11.png "node 11")
+
+Collection - a collection of your choice
+
+Permissions - full access
+
+Payload:
+```js
+{
+    "calendar_event_id": "{{$trigger.body.data.calendar_event_id}}",
+    "calendar_event_start": "{{$trigger.body.data.calendar_event_start}}",
+    "calendar_event_end": "{{$trigger.body.data.calendar_event_end}}",
+    "name": "{{$trigger.body.data.name}}",
+    "description": "{{$trigger.body.data.description}}"
+}
+```
+
+
+***
+
+- Node 12 - "Transform Payload"
+
+![node 12](/copy-this-template-directory/directus_flow_4_12.png "node 12")
+
+if there was an error during operation. This node is not necessary
+
+
+After you save this Flow, copy the resulting webhook URL somewhere.
+
+***
+
+&nbsp; 
+
+
+## Setup in Google side
+
+[Google Apps Script](https://developers.google.com/apps-script/overview) is usually enabled by default, but if you are using Google Workspace within the organization, your Admin might disable Google Apps Script.
+
+Google Apps Script can be created as a dedicated file (Drive → New → More → Google Apps Script).
+
+But for this project, we will use Script within Spreadsheet. That way it is easier to use a spreadsheet for config and for logging.
+
+For quicker setup, please use [this spreadsheet template copy](https://docs.google.com/spreadsheets/d/1mXuSVR01uueeRDTjNeSXFH2rAziWkXdrNb2Jym413Fw/copy), it has full script inside and sheet with config.
+
+In your spreadsheet copy, open the menu Extensions → Apps Script. It will open Script Editor.
+
+![GAS 04](/copy-this-template-directory/GAS_04.png "GAS 04")
+
+Click the button `Deploy` → `New Deployment`.
+
+When Google Apps Script is deployed as Web App, it creates a unique URL like `https://script.google.com/macros/s/xxxx/exec`.
+When this URL is called with a GET request, the script function `doGet` is executed.  When this URL is called with a POST request, script function `doPost` is executed. We will use POST requests and doPost function.
+ 
+ ***
+
+![GAS 05](/copy-this-template-directory/GAS_05.png "GAS 05")
+
+set type to web app. Description - anything. Execute as - Me. Who has access - anyone.
+Click `Deploy`.
+
```

Comment:
Write as prose - right now it's just a bunch of tiny points. Drop the image - I don't think the illustration is needed here.